gomint-server/src/main/java/io/gomint/server/network/packet/PacketPlayerlist.java

```diff
@@ -56,17 +66,24 @@ public void serialize( PacketBuffer buffer, int protocolID ) {
                 buffer.writeUnsignedVarInt( entry.skin.getGeometryData().length() );
                 buffer.writeBytes( entry.skin.getGeometryData().getBytes() );
 
-                // xbox user id
-                buffer.writeString( entry.xboxId );
+                // Is teacher
+                buffer.writeBoolean( false );
 
-                // TODO: Is this the same as the unknown one in SpawnPlayer?
-                buffer.writeString( entry.uuid.toString() );
+                // Is host
+                buffer.writeBoolean( false );
             }
         } else {
             for ( Entry entry : this.entries ) {
                 buffer.writeUUID( entry.getUuid() );
             }
         }
+
+        if ( mode == 0 ) {
+            for (int i = 0; i < entries.size(); i++) {
+                // Is skin trusted
+                buffer.writeBoolean( true );
```

Comment:
Is the logic behind this known? What is a "trusted" skin?

Reply:
I've heard tuere is an optionin thr client to only show trusted skins. Probably only buyable skins are trusted, while custom ones are not.

Reply:
Well, @Janmm14 is true and for now this is a static value because our skin handling is still to do, because this boolean value can be read from the login packet and then we have to send it there